lib/request.js

```diff
@@ -280,6 +280,7 @@ module.exports = {
    */
 
   get URL() {
+    /* istanbul ignore else */
```

Comment:
i think it is useless here?

Reply:
Istanbul makes `else` as a default branch, even though you don't write `else`. So coverage test will cover not-exist `else` as coverage